lib/mongo_client.js

```diff
@@ -147,11 +147,11 @@ inherits(MongoClient, EventEmitter);
  */
 
 /**
- * Connect to MongoDB using a url as documented at
+ * Connect to MongoDB using a URL as documented at
```

Comment:
I'm fine with the correction to the JSDOC, but I'd like to leave the text changes out for now. Let's correct the issue with intellisense, and worry about the grammar later.

Reply:
OK, can you please cherry-pick?

Reply:
Done, but I don't guarantee I'll come back to this doc to fix the grammar.

Reply:
Thanks @dandv, we have work slated shortly for a near complete rewrite of our documentation. The grammar should be corrected in that process.